presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -269,6 +269,13 @@ void PrestoServer::run() {
     }
   }
 
+  LOG(INFO) << "STARTUP: Driver CPU executor has "
+            << driverCPUExecutor()->numThreads() << " threads.";
+  if (httpServer_->getExecutor()) {
+    LOG(INFO) << "STARTUP: HTTP Server executor has "
+              << httpServer_->getExecutor()->numThreads() << " threads.";
+  }
```

Comment:
Also log the number of threads in spilling executor?

Reply:
Where do we create it?